src/app/components/list/list-action-button.component.html

```diff
@@ -7,6 +7,9 @@
     <p>
       Lists display data in a consistent, flexible way.
     </p>
+    <sky-alert alertType="warning">
+      We recommend using the <a href="https://developer.blackbaud.com/skyux-data-manager/docs/data-manager">data manager component</a> with the <a href="https://developer.blackbaud.com/skyux/components/data-entry-grid">data entry grid component</a> or the <a href="https://developer.blackbaud.com/skyux-lists/docs/repeater">repeater component</a> instead of lists because they extend the functionality and improve the behavior. We will deprecate the list module in a future version of SKY UX.
```

Comment:
```suggestion
      We recommend using the <a href="https://developer.blackbaud.com/skyux-data-manager/docs/data-manager">data manager</a>, <a href="https://developer.blackbaud.com/skyux/components/data-entry-grid">data entry grid</a>, or <a href="https://developer.blackbaud.com/skyux-lists/docs/repeater">repeater</a> components instead of lists because they extend the functionality and improve the behavior. We will deprecate the list module in a future version of SKY UX.
```
Tweaking the way we list the components for consistency with the existing notes, which don't indicate a combination of data manager and the other components. If we do want to recommend that combination, we need to change `or` to `with` in the notes at  https://developer.blackbaud.com/skyux-list-builder-view-grids/docs/list-view-grid?docs-active-tab=design and https://developer.blackbaud.com/skyux-list-builder-view-checklist/docs/list-view-checklist?docs-active-tab=design.